.github/workflows/microk8s-tests.yml

```diff
@@ -0,0 +1,196 @@
+name: Test Kubeflow
+
+on:
+  push:
+    paths-ignore:
+      - 'acceptancetests/**'
+      - 'doc/**'
+      - 'snap/**'
+      - 'testcharms/**'
+      - 'testing/**'
+      - 'tests/**'
+  pull_request:
+    paths-ignore:
+      - 'acceptancetests/**'
+      - 'doc/**'
+      - 'snap/**'
+      - 'testcharms/**'
+      - 'testing/**'
+      - 'tests/**'
```

Comment:
Nice, can we add these to the client tests and smoke tests as a driveby fix

Reply:
sure